backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -530,4 +475,23 @@
       end
     end
   end
+
+  context 'with only read permissions' do
+    stub_authorization!(permission_sets: [Spree::PermissionSets::OrderDisplay])
+
+    it "should not display forbidden links" do
```

Comment:
My argument is: Entropy in the system should be reduced. The less entropy on ANY level: Semantics, Syntax, Data the better.

Will followup in a separate discussion.